src/Toolbar/Toolbar.react.js

```diff
@@ -434,13 +473,17 @@ class Toolbar extends PureComponent {
                     onLeftElementPress={onLeftElementPress}
                     isSearchActive={isSearchActive}
                     onSearchClose={this.onSearchCloseRequested}
+                    accessibilityLabel={leftElementAccessibilityLabel}
+                    accessibilityTraits={leftElementAccessibilityTraits}
                 />
                 <CenterElement
                     {...this.props}
                     onPress={onPress}
                     searchValue={searchValue}
                     isSearchActive={isSearchActive}
                     onSearchTextChange={this.onSearchTextChanged}
+                    accessibilityLabel={centerElementAccessibilityLabel}
+                    accessibilityTraits={centerElementAccessibilityTraits}
```

Comment:
Could we set up those traits automatically by a type of `centerElement`, `leftElement` and `rightElement`?